src/appleseed/renderer/kernel/lighting/drt/drtlightingengine.cpp

```diff
@@ -125,15 +127,17 @@ namespace
             {
                 RENDERER_LOG_INFO(
                     "distribution ray tracing settings:\n"
-                    "  ibl              %s\n"
-                    "  max path length  %s\n"
-                    "  rr min path len. %s\n"
-                    "  dl light samples %s\n"
-                    "  ibl env samples  %s",
+                    "  ibl                     %s\n"
```

Comment:
These messages are all indented similarly such that they line up nicely in the log output.

I completely agree that the space for the label is way too small, and we'll fix that as some point. In the meantime, I suggest leaving the indentation as it was and using a shorter label for the light threshold, maybe `dl light thresh.`?